diff --git a/irssi/src/core/expandos.c b/irssi/src/core/expandos.c
index 33c7039..629a2ff 100644
--- a/irssi/src/core/expandos.c
+++ b/irssi/src/core/expandos.c
@@ -18,6 +18,9 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */

+#include <glib.h>
+#include <glib/gprintf.h>
+
 #include "core.h"
 #include "module.h"
 #include "modules.h"
@@ -37,6 +40,10 @@
 #  include <sys/utsname.h>
 #endif

+#ifndef IRSSI_VERSION_DATE
+  #define IRSSI_VERSION_DATE 20130525
+#endif
+
 #define MAX_EXPANDO_SIGNALS 10

 typedef struct {
@@ -404,7 +411,9 @@ static char *expando_target(SERVER_REC *server, void *item,
 static char *expando_releasedate(SERVER_REC *server, void *item, int *free_ret
 {
         *free_ret = TRUE;
-       return g_strdup_printf("%d", IRSSI_VERSION_DATE);
+        int version = 20130525;
+        const gchar ret = g_strdup_printf("%d", version);
+        return ret;
 }

 /* client release time (in HHMM format) */
